pkg/utils/jwt.go

```diff
@@ -0,0 +1,34 @@
+package utils
+
+import (
+	"fmt"
+	"github.com/golang-jwt/jwt/v4"
+)
+
+// GetFieldFromJWT Returns the value of a field from a given JWT.
+//
+// WARNING: This function does not verify the token.
+//
+// Only use this function in cases where you know the signature is valid and want to extract values from it.
```

Comment:
Since this function is a refactor, any reason to not also refactor from ParseUnverified to Parse?